frontend/src/components/header/header.module.scss

```diff
@@ -0,0 +1,20 @@
+body {
+  background: #f1f4f6;
+}
+
+.header {
+  position: relative;
+  z-index: 2;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 105px;
```

Comment:
Usually, the height of the header is considered as the project CSS variable. Because it can be reused in multiple places to properly calculate the size or the offsets. So I would recommend to put it in the root of the styles and then use as CSS variable